### PR TITLE
Add Endnote XML file reader

### PR DESCRIPTION
**Pull request by rohitgarud**
Adds Endnote XML file reader which can be used to import XML files generated using Endnote reference manager. There are many open source systematic review datasets available in Endnote xml format, which can be added to the dataset repository

### CODE REVIEW

**Comment by rohitgarud** [comment]
Is there a particular reason for not implementing this? I think it is a reasonable thing to do to track the records.
https://github.com/asreview/asreview/blob/9064aa61bf89721a9e56e1c49291c3782b4e7089/asreview/io/utils.py#L157-L169

File: asreview/io/xml_reader.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import pandas as pd
+
+from asreview.io.utils import _standardize_dataframe
+
+
+class EndnoteXMLReader:
+    """Endnote XML file reader."""
+
+    read_format = [".xml"]
+    write_format = [".csv", ".tsv", ".xlsx"]
```
Comment (J535D165) [inline asreview/io/xml_reader.py:12]:
```suggestion
    write_format = [".csv", ".tsv", ".xlsx", ".ris"]
```

@terrymyc what does it imply to add RIS here?

**Comment by PeterLombaers** [comment]
@rohitgarud We've changed in #1591 how the frontend finds the available data reader classes. This means that a custom data reader like the one of this pull request should be easy to implement as an ASReview extension, just like you can make you own model extension. I'll close this pull request because we think that the reader is a bit too specific to include in the base code of ASReview, but I hope you can make it work as an extension, because it does look useful!

**Comment by rohitgarud** [comment]
Sure.. makes sense.. will try to develop an extension for this reader

**Comment by J535D165** [comment]
Would be great @rohitgarud. And sorry it took so long. We are always very grateful for your contributions.

**Comment by rohitgarud** [comment]
No worries.. I was just exploring things at that time.. I want to start contributing again but not getting time due to other commitments..

**Pull request closed without merging**